mgmtsystem/openshift.py

```diff
@@ -67,8 +67,8 @@ def list_service(self):
             entities.append(entity)
         return entities
 
-    def list_image_registry(self):
-        """Returns list of image registries (image streams)"""
+    def list_docker_registry(self):
```

Comment:
@mfalesni It won't, cause list_docker_registry returns the ip and the port of the docker registry, while list_image_registry in Kubernetes class returns

[ImageRegistry(host=u'openshift3', port=''),
 ImageRegistry(host=u'registry.access.redhat.com', port=''),
 ImageRegistry(host=u'docker.io', port=''),
 ImageRegistry(host=u'brew-pulp-docker01.web.prod.ext.phx2.redhat.com', port=u'8888')]

which is the number of registries